### ISSUE TITLE
"Error while logging in:  Exiting."

### ISSUE THREAD

**Issue by NichSchlagen** [stecklars/dynamic-dns-netcup-api#11]
Hallo,

jedes Mal, wenn ich das Script ausführe bekomme ich diesen Fehler:

*"[ERROR] Error while logging in:  Exiting."*

Die Customer-Nummer ist korrekt, der API Key und das Passwort auch.
Woran kann es liegen?

Mit freundlichen Grüßen
Tim

**Comment by seanlucakrueger** [comment]
Moin,
bei mir kommt auch so eine Fehlermeldung. Woran könnte das liegen? Ich mach ja nur eine Request oder?
Genaue Fehlermeldung:
___________________________

[2021/08/22 21:09:56 +0200][ERROR] Error while logging in: More than 180 requests per minute. Please wait and retry later. Please contact our customer service to find out if the limitation of requests can be increased. Exiting.
___________________________

Mit freundlichen Grüßen
Sean Luca

**Comment by tojo2503** [comment]
Hi,

I can second @NichtSchlagen.
I also get the error: 

"[ERROR] Error while logging in: Exiting."

Nothing changed on my side and double checked API password and API key.
Did netcup change something?

Thanks a lot for your work!

edit: sorry. That was my mistake. Something was wrong with my DNS service on my server, therefore I got the error. Everything is working as expected now. Thanks!

**Comment by stecklars** [comment]
@seanlucakrueger Der Fehler resultiert ziemlich sicher aus falscher Kundennummer / API-Key / API-Passwort. Bitte das nochmal gründlich prüfen.

@tojo2503 Good to know that you found a solution.

@NichSchlagen Did you also find a solution so far? Otherwise, please add

`var_dump($result);`

before line 139 in functions.php (`outputStderr(sprintf("Error while logging in: %s Exiting.", $result['longmessage']));`) so that it looks like this:

```
var_dump($result);
outputStderr(sprintf("Error while logging in: %s Exiting.", $result['longmessage']));
```

Then run update.php again, you should get more (debugging) output. Is the error visible there?

**Comment by xXValiXx** [comment]
@stecklars 

I tested it today with that error. I added the var_dump and its saying "NULL"

![image](https://user-images.githubusercontent.com/47116168/132255142-d1b3e6db-293a-4b6e-b0e0-5fceb836f481.png)

Problem Solved!

The Docker Container i have used, has no real time. So the Script does not work correctly.

My bad Sorry.

**Comment by stecklars** [comment]
Error messages for cURL / HTTP errors will now be displayed (fixed in commit 6a992c475b4a061937f5936560cebd066f42d536)
